.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule ".ci"]
 	path = .ci
 	url = https://github.com/m3db/ci-scripts.git
+[submodule "site/site/docs-theme"]
```

Comment:
@gibbscullen None of this theme stuff should be here… Your local git might need some changes, not sure. 🤔

Reply:
@ChrisChinchilla do I need to delete the existing theme in lieu of the new theme?